docs/wire_compiler.md

```diff
@@ -535,9 +535,9 @@ wire {
 }
 ```
 
- [SchemaHandler.Factory]: https://github.com/square/wire/blob/fd0a00ff5b6033ed9d8c2d392ff06338467a026f/wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/SchemaHandler.kt#L192-L194
- [SchemaHandlerRecipes]: https://github.com/square/wire/blob/c3c5f559556ad9d41582a0e0a025679b5493f7aa/wire-library/wire-schema-tests/src/test/java/com/squareup/wire/recipes
- [SchemaHandler]: https://github.com/square/wire/blob/fd0a00ff5b6033ed9d8c2d392ff06338467a026f/wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/SchemaHandler.kt#L24
+ [SchemaHandler.Factory]: https://github.com/square/wire/blob/master/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/SchemaHandler.kt#L192
+ [SchemaHandlerRecipes]: https://github.com/square/wire/tree/master/wire-schema-tests/src/test/java/com/squareup/wire/recipes
+ [SchemaHandler]: https://github.com/square/wire/blob/master/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/SchemaHandler.kt
```

Comment:
When looking at the main branch, in github, you can tap `y` and the url will change to a permalink one. I've done that with the 3 urls.
```suggestion
 [SchemaHandler.Factory]: https://github.com/square/wire/blob/9d0fcfce386cc964fe30521c017af520de59c7df/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/SchemaHandler.kt#L192-L194
 [SchemaHandlerRecipes]: https://github.com/square/wire/tree/9d0fcfce386cc964fe30521c017af520de59c7df/wire-schema-tests/src/test/java/com/squareup/wire/recipes
 [SchemaHandler]: https://github.com/square/wire/blob/9d0fcfce386cc964fe30521c017af520de59c7df/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/SchemaHandler.kt
```